packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/base-types.ts

```diff
@@ -496,6 +496,300 @@ export enum CompressionType {
   GZIP = 'Gzip'
 }
 
+//
+// Create Processing Job types
+//
+
+/**
+ * Configures the processing job to run a specified Docker container image.
+ *
+ * @experimental
+ */
+export interface AppSpecification {
+  /**
+   * The arguments for a container used to run a processing job.
+   *
+   * @default None
+   */
+  readonly containerArguments?: string[];
+
+  /**
+   * The entrypoint for a container used to run a processing job.
+   *
+   * @default None
```

Comment:
(Same as similar comment above):

I decided that it instead might be more helpful to link to https://docs.aws.amazon.com/sagemaker/latest/dg/build-your-own-processing-container.html since there's some nuance involved.

To whoever ends up reviewing this PR, let me know if you would suggest otherwise